### PR TITLE
Fix News Item and Link content type (#15&#19)

### PR DESCRIPTION
**Pull request by iRohitSingh**
LINK : 

Screenshot 2023-07-15 at 22 46 53

NEWS ITEM : 

Screenshot 2023-07-15 at 22 49 27

mobile: 

Screenshot 2023-07-15 at 22 50 17

### CODE REVIEW

**Comment by iRohitSingh** [comment]
news items example content : pr https://github.com/kitconcept/bfs-intranet/pull/176

**Review by steffenri — APPROVED** [review]

**Comment by steffenri** [comment]
@iRohitSingh Please fix merge conflicts. Then you can merge.

File: src/theme/_content.scss [hunk 1]
```diff
@@ -9,6 +9,9 @@
 .contenttype-news-item {
   .documentFirstHeading {
     margin-top: 0px;
+    @media only screen and (max-width: 769px) {
```
Comment (sneridagh) [inline src/theme/_content.scss:12]: @iRohitSingh no bare media queries unless it's strictly necessary, please. There's a variable for that.

File: src/theme/_content.scss [hunk 2]
```diff
@@ -19,14 +22,30 @@
     margin-right: auto;
     margin-bottom: 40px;
     margin-left: auto;
+    @media only screen and (max-width: 769px) {
```
Comment (sneridagh) [inline src/theme/_content.scss:25]: Same

**Comment by sneridagh** [comment]
@iRohitSingh the CSS included in this one is not ok.

**Pull request closed without merging**